components/IssueCredentialSuccess.yml

```diff
@@ -0,0 +1,34 @@
+openapi: 3.0.0
+info:
+  version: "0.0.3-unstable"
+  title: VC API
+  description: This is an Experimental Open API Specification for the [VC Data Model](https://www.w3.org/TR/vc-data-model/).
+  license:
+    name: W3C Software and Document License
+    url: http://www.w3.org/Consortium/Legal/copyright-software.
+  contact:
+    name: GitHub Source Code
+    url: https://github.com/w3c-ccg/vc-api
+paths:
+components:
+  schemas:
+    IssueCredentialSuccess:
+      type: object
+      description: The format returned from a successful issue request.
+      properties:
+        VerifiableCredential:
+          type: object
+          description: A JSON-LD Verifiable Credential with a proof.
+          allOf:
+            - $ref: "./Credential.yml#/components/schemas/Credential"
+            - type: object
+              properties:
+                proof:
+                  $ref: "./DataIntegrityProof.yml#/components/schemas/DataIntegrityProof"
+        IssuanceMetadata:
+          type: object
+          description: Metadata about the issued credential.
+          properties:
+            credentialId:
+              type: string
+              description: An identifier for the credential. If credentialId is not set by the user in the issue request, credential.id will be duplicated here.
```

Comment:
```suggestion
```

The group discussed this and decided to not define the metadata field at this point in time.